src/algorithm/joint-configuration.hpp

```diff
@@ -303,16 +315,14 @@ namespace se3
     template<typename JointModel>
     static void algo(const se3::JointModelBase<JointModel> & jmodel,
                      se3::JointDataBase<typename JointModel::JointData> &,
-                     const se3::Model & model,
+                     const se3::Model & ,
```

Comment:
This not a problem, you just need to create something like JOINT_MODEL_VISITOR_INIT